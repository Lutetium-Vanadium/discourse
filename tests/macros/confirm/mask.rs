fn main() {
    let q = inquisition::questions![Confirm { mask: todo!() }];
}
