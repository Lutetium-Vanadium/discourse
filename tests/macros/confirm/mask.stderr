error: option `mask` does not exist for kind `confirm`
 --> $DIR/mask.rs:2:47
  |
2 |     let q = inquisition::questions![Confirm { mask: todo!() }];
  |                                               ^^^^
